backend/public-graph/graph/resolver.go

```diff
@@ -2702,19 +2702,17 @@ func (r *Resolver) ProcessPayload(ctx context.Context, sessionSecureID string, e
 			tracer.ResourceName("go.unmarshal.messages"), tracer.Tag("project_id", projectID))
 		defer unmarshalMessagesSpan.Finish()
 
-		if projectID == 1 {
-			logRows, err := clickhouse.ParseConsoleMessages(projectID, sessionSecureID, messages)
-			if err != nil {
-				log.WithError(err).Error("failed to parse console messages")
-			} else {
-				if err := r.BatchedQueue.Submit(&kafka_queue.Message{
-					Type: kafka_queue.PushLogs,
-					PushLogs: &kafka_queue.PushLogsArgs{
-						SessionSecureID: sessionSecureID,
-						LogRows:         logRows,
-					}}, sessionSecureID); err != nil {
-					log.WithError(err).Error("error writing console messages to clickhouse")
-				}
+		logRows, err := clickhouse.ParseConsoleMessages(projectID, sessionSecureID, messages)
+		if err != nil {
+			log.WithError(err).Error("failed to parse console messages")
+		} else {
+			if err := r.BatchedQueue.Submit(&kafka_queue.Message{
+				Type: kafka_queue.PushLogs,
+				PushLogs: &kafka_queue.PushLogsArgs{
+					SessionSecureID: sessionSecureID,
+					LogRows:         logRows,
+				}}, sessionSecureID); err != nil {
+				log.WithError(err).Error("error writing console messages to clickhouse")
```

Comment:
in a future pr, could we move this into the public graph PushPayload handler since we are no longer depending on the project id lookup?

Reply:
I might be misunderstanding what you're suggesting but we are still using the project id (see the first param to `ParseConsoleMessages`).